fabric-content-registries-v0/src/main/java/net/fabricmc/fabric/api/registry/CompostingChanceRegistry.java

```diff
@@ -22,7 +22,9 @@
 /**
  * Registry of items to 0.0-1.0 values, defining the chance of a given item
  * increasing the Composter block's level.
+ * @deprecated replaced by {@link net.fabricmc.fabric.api.content.registry.v1.CompostableItemRegistry}
  */
+@Deprecated
 public interface CompostingChanceRegistry extends Item2ObjectMap<Float> {
 	CompostingChanceRegistry INSTANCE = new CompostingChanceRegistryImpl();
```

Comment:
Would it be better to have this field changed to the instance of `CompostableItemRegistry`?

Reply:
explained in a reply to modmuss that these old APIs have an extra method that is not implemented in the new api, making it pretty much impossible to move over (explanation on removal in PR description)

Reply:
oh right, `clear()` got removed